include/clink/feature/one_hot_encoder.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ONE_HOT_ENCODER_H_
+#define ONE_HOT_ENCODER_H_
+
+#include "../../../protos/OneHotEncoder.pb.h"
+#include "clink/linalg/sparse_vector.h"
+
+namespace clink {
+
+class OneHotEncoderModel {
+  public:
+    OneHotEncoderParams params;
```

Comment:
Could you add empty lines as appropriate?

It is probably better to mark member variables and methods `private` as much as possible. Could you check this?

And we usually put member variables after methods in the code. Could we follow TFRT's code style?

And could we have doc for public methods here (unless the method's semantic is obvious based on its name)?

Reply:
OK. I'll reformat the codes.